File: app/src/main/java/dev/olog/basil/domain/entity/Recipe.java
package dev.olog.basil.domain.entity;

public class Recipe {

    private final long id;
    private final String name;

    public Recipe(long id, String name) {
        this.id = id;
        this.name = name;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/dev/olog/basil/domain/entity/Recipe.kt
package dev.olog.basil.domain.entity

data class Recipe(
        val id: Long,
        val name: String,
        val description: String,
        val people: Int,
        val calories: Int,
        val ingredients: MutableList<Ingredient>,
        val images: MutableList<String>,
        val tags: MutableList<Tag>
) {

    fun addImage(imageUri: String){
        this.images.add(imageUri)
    }

    fun addTag(tag: Tag) {
        this.tags.add(tag)
    }

    fun addIngredient(ingredient: Ingredient) {
        this.ingredients.add(ingredient)
    }

}
